pom.xml

```diff
@@ -169,14 +169,14 @@ POSSIBILITY OF SUCH DAMAGE.
           <skipDeploy>true</skipDeploy>
         </configuration>
       </plugin>
-      <plugin>
+      <!--plugin>
```

Comment:
Rather than comment-out lines, just nuke 'em! (If you really want to keep them, then add a comment explaining why.)

Reply:
updated